app/MacPorts/settings.py

```diff
@@ -27,9 +27,7 @@
 DEBUG = True
 
 ALLOWED_HOSTS = [
-    'frozen-falls-98471.herokuapp.com',
-    'localhost',
-    '127.0.0.1',
+    '*'
```

Comment:
Looking for feedback from @rajdeepbharati.

Reply:
Do I need to set up a postgres server?

Reply:
I am getting this error:
![Screenshot 2019-07-12 at 2 53 03 PM](https://user-images.githubusercontent.com/30660439/61117745-d1a79480-a4b4-11e9-9932-4cca554edc1f.png)

Reply:
Sorry for late reply. You are probably trying to connect to the database running outside docker container and providing the host `localhost`. A docker container does not have access to the host machine's database through localhost.

I have never tried this, but on Docker for Mac, instead of `localhost` you can use `host.docker.internal` in ` DB_HOST` to access the host's database.

On Docker for Linux, you can use the argument `--network="host"` while running the container.

Reply:
Thank you, it works using `host.docker.internal`. Sorry I'm not an expert in Docker

Reply:
When I run `docker exec c52e7c755761 python3 /code/app/manage.py load` it gives this error:
`CommandError: "portindex.json" not found. Make sure "portindex.json" is a valid JSON file and is in the root of the project` although I have placed `portindex.json` in `app` directory

Reply:
Did you pull the image from docker hub? If yes, you can use the command `load-rsync` instead of `load`.

I am sorry for improper instructions, the setup is being changed and improved regularly, but I will write proper setup instructions soon. For now, using `load-rsync` will solve this.

Reply:
Another better way to do this would be:

```
docker exec <containerid> supervisorctl start app-load
```
Then you wont be worry about the command being running.

But you should pull the image from docker hub.

Reply:
Cool! Does the rsync daemon need to be kept running?

Reply:
Thanks 😄

Reply:
How did you start the command? If you ran

```
docker exec <id> python3 /code/app/manage.py load-rsync
```
then you would have to keep the command running. It should run for 15-20 minutes, there will be some warnings about dependencies, you can ignore them.

But if you use
```
docker exec <id> supervisorctl start app-load
```

then you won't have to worry about anything.

Reply:
I used this: `docker exec <id> python3 /code/app/manage.py load-rsync`

Reply:
Okay, the command should have finished by now.